synthesizer/event.cpp

```diff
@@ -356,8 +356,9 @@ QString midiMetaName(int meta)
       return QString(s);
       }
 
+// We don't support negative pitch, but Midi does. Let's center by adding/adjusting by 8192.
 int midiBendPitch(int pitch) {
-      return (pitch * 8192) / (PITCH_BEND_SENSITIVITY * 100) + 8192;
+      return (pitch * 8192 * 2) / (PITCH_BEND_SENSITIVITY * 100) + 8192;
```

Comment:
what does the "2" means?

Reply:
Twice 8192, 8192 is essentially the midpoint of the range that we're using and so the full range would be +/- 8192 which combined is 16384. Having a magic number here isn't great, so I'd like to keep it down to one magic number rather than 2.

Reply:
Why not make 8192 a constant variable (`constexpr static int pitchRadius/*or some better name*/ 8192;`)?